src/lib/components/phoneInput/__snapshots__/phoneInput.test.js.snap

```diff
@@ -5,7 +5,7 @@ exports[`PhoneInput renders properly 1`] = `
   className=" theme-purple inputWrapper"
 >
   <div
-    className="react-phone-number-input input"
+    className="react-phone-number-input input PEPITOO"
```

Comment:
I am not entirely sure we should merge this to master :D .

Reply:
haha, interesting class